addons/binding/org.openhab.binding.lgtvserial/ESH-INF/thing/thing-types-M6503C.xml

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="lgtvserial"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Unimplemented so far : Abnormal state, Remote lock, Auto configure, command 36,37,38,39,42 (timers), reset, 52 scheduling input select -->
+    <thing-type id="lgtv-M6503C">
+        <label>M6503C model</label>
+        <description><![CDATA[This thing supports the M6503C LG TV]]></description>
+
+        <channels>
+<!--             <channel id="" typeId=""/> -->
+            <channel id="aspect-ratio" typeId="aspect-ratio-M6503C"/>
+            <channel id="auto-sleep" typeId="auto-sleep"/>
+            <channel id="auto-volume" typeId="auto-volume"/>
+            <channel id="balance" typeId="balance"/>
+            <channel id="brightness" typeId="brightness"/>
+            <channel id="contrast" typeId="contrast"/>
+            <channel id="color" typeId="color"/>
+            <channel id="color-temperature" typeId="color-temperature"/>
+            <channel id="elapsed-time" typeId="elapsed-time"/>
+            <channel id="fan-fault-check" typeId="fan-fault-check"/>
+            <channel id="h-position" typeId="h-position"/>
+            <channel id="h-size" typeId="h-size"/>
+            <channel id="input" typeId="input-M6503C"/>
+            <channel id="input2" typeId="input2-M6503C"/>
+            <channel id="ir-key-code" typeId="ir-key-code-M6503C" />
+            <channel id="ism-method" typeId="ism-M6503C"/>
+            <channel id="key" typeId="ir-key-codes-M6503C"/>
+            <channel id="lamp-fault-check" typeId="lamp-fault-check"/>
+            <channel id="natural-mode" typeId="natural-mode"/>
+            <channel id="osd-select" typeId="osd-select"/>
+            <channel id="picture-mode" typeId="picture-mode"/>
+            <channel id="power" typeId="power"/>
+            <channel id="power-indicator" typeId="power-indicator"/>
+            <channel id="power-saving" typeId="power-saving"/>
+            <channel id="raw" typeId="raw"/>
+            <channel id="screen-mute" typeId="screen-mute"/>
+            <channel id="serial-number" typeId="serial-number"/>
+            <channel id="sharpness" typeId="sharpness"/>
+            <channel id="sleep-time" typeId="sleep-time"/>
+            <channel id="software-version" typeId="software-version"/>
+            <channel id="sound-mode" typeId="sound-mode-M6503C"/>
+            <channel id="speaker" typeId="speaker"/>
+            <channel id="temperature-value" typeId="temperature-value"/>
+            <channel id="tile" typeId="tile-M6503C"/>
```

Comment:
tile was only found in the M6503C, the model I have doesn't reference it so it's definition is in thing-types-M6503C.xml. The other 2 have been forgotten.